### PR TITLE
FB8-99: test: slocket_listen.py /bin/env -> /usr/bin/env

### PR DESCRIPTION
**Pull request by inikep**
Summary:

Jira issue: https://jira.percona.com/browse/FB8-99

Reference Patch: https://github.com/facebook/mysql-5.6/commit/37e27fa
Reference Patch: https://github.com/facebook/mysql-5.6/commit/0e272e8
Reference Patch: https://github.com/facebook/mysql-5.6/commit/cc4c998

Various small fixes for slocket_listen.py script.

Squash with D6792354

### CODE REVIEW

File: sql/log.cc [hunk 1]
```diff
@@ -1637,6 +1637,10 @@ bool write_log_to_socket(int sockfd, THD *thd, ulonglong end_utime) {
                     local.tm_year % 100, local.tm_mon + 1, local.tm_mday,
                     local.tm_hour, local.tm_min, local.tm_sec);
 
+  if (len < buf_sz)
+    len += snprintf(buf + len, buf_sz - len, "# Threadid: %lu \n",
+                    (ulong)thd->thread_id());
```
Comment (percona-ysorokin) [inline sql/log.cc]:
```suggestion
                    static_cast<ulong>(thd->thread_id()));
```

**Comment by inikep** [comment]
PR was updated

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Comment by inikep** [comment]
The commit was rebased.

**Review by hermanlee — APPROVED** [review]

**Pull request closed without merging**